### PR TITLE
feat/nav layout change

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

**Comment by MildTomato** [comment]
Screenshot 2023-07-24 at 12 25 20 PM

onSelect not working for "new project" or "new organization"

**Comment by MildTomato** [comment]
Screenshot 2023-07-24 at 12 26 34 PM

a11y issue. keyboard nav impossible to change theme

**Comment by MildTomato** [comment]
Screenshot 2023-07-24 at 12 27 02 PM

long string/truncate issue with email

**Comment by MildTomato** [comment]
Screenshot 2023-07-24 at 12 27 50 PM

a11y issue. keyboard nav "onSelect" not working.

**Comment by MildTomato** [comment]
Screenshot 2023-07-24 at 12 29 09 PM

issue with wrong project/org highlighted which is a bit distracting.
think we should invest some time in an 'active' state and make sure the active project/org is highlighted by default when opened.

**Comment by MildTomato** [comment]
Screenshot 2023-07-24 at 12 31 08 PM

tailwind issue. 
remove the default browser focus state for ComboBox input

**Comment by MildTomato** [comment]
Screenshot 2023-07-24 at 12 31 59 PM

also need to reduce text size to `text-sm` like rest of the app.

File: studio/.gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 .env.production.local
 
 .vercel
+
+/public/dashboard
```
Comment (joshenlim) [inline studio/.gitignore:37]:
```suggestion
```

File: studio/.gitignore [hunk 2]
```diff
@@ -33,3 +33,4 @@ yarn-error.log*
 .env.production.local
 
 .vercel
+
```
Comment (joshenlim) [inline studio/.gitignore:36]:
```suggestion
```

File: studio/.gitignore (hunk 1 above)
Comment (MildTomato) [reply studio/.gitignore]: thanks !

Comment (alaister) [reply studio/.gitignore]: @MildTomato didn't you know @joshenlim is telepathic?

Comment (MildTomato) [reply studio/.gitignore]: this was weirdly creepy

File: studio/hooks/misc/useSelectedOrganization.ts [hunk 3]
```diff
@@ -3,15 +3,23 @@ import { useOrganizationsQuery } from 'data/organizations/organizations-query'
...
+import { LOCAL_STORAGE_KEYS } from 'lib/constants'
 
 export function useSelectedOrganization({ enabled = true } = {}) {
   const { slug } = useParams()
   const { data } = useOrganizationsQuery({ enabled })
   const selectedProject = useSelectedProject()
+  const localStorageSlug =
+    typeof window !== 'undefined'
+      ? localStorage.getItem(LOCAL_STORAGE_KEYS.RECENTLY_VISITED_ORGANIZATION)
+      : undefined
```
Comment (alaister) [inline studio/hooks/misc/useSelectedOrganization.ts:15]: Do we want to consider caching this in a valtio store to prevent slowing down react's rendering?

**Review by alaister — APPROVED** [review]
LGTM! Maybe a small improvement:

File: studio/hooks/misc/useSelectedOrganization.ts (hunk 3 above)
Comment (joshenlim) [reply studio/hooks/misc/useSelectedOrganization.ts:15]: just to check though cause useSelectedOrganization already returns a useMemo - would this already mitigate the performance issue here?

**Review by alaister — APPROVED** [review]

**Pull request merged**